### PR TITLE
Onboard ibm-mongodb operator

### PR DESCRIPTION
**Pull request by DanielXLee**

### CODE REVIEW

File: deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml [hunk 1]
```diff
@@ -11,3 +11,6 @@ spec:
   - name: ibm-licensing-operator
     spec:
       IBMLicensing: {}
+  - name: ibm-mongodb-operator
+    spec:
+      replicas: 3
```
Comment (chenzhiwei) [inline deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml]: should be:

```
spec:
    mongodb: {}
```

File: deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -57,6 +68,12 @@ metadata:
                 "spec": {
                   "IBMLicensing": {}
                 }
+              },
+              {
+                "name": "ibm-mongodb-operator",
+                "spec": {
+                  "replicas": 3
```
Comment (chenzhiwei) [inline deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml]: here

File: deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml [hunk 3]
```diff
@@ -20,3 +20,11 @@ spec:
     sourceNamespace: openshift-marketplace
     targetNamespaces:
       - ibm-licensing-operator
+  - name: ibm-mongodb-operator
+    namespace: ibm-mongodb-operator
+    channel: alpha
+    packageName: ibm-mongodb-operator-app
+    sourceName: opencloud-operators
+    sourceNamespace: openshift-marketplace
+    targetNamespaces:
```
Comment (gyliu513) [inline deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml]: I think we need to update the example here by adding `description` field and remove `targetNamespaces` for all operators in `MetaOperatorCatalog`

Comment (DanielXLee) [reply deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml]: 👌， I'll update all the onboarded operator

**Pull request merged**